src/Loading/babylon.sceneLoader.ts

```diff
@@ -189,6 +189,82 @@
             }
         }
 
+        private static _CleanMatricesIndices(meshes:AbstractMesh[], skeletons:Skeleton[]): void {

```

Comment:
I would like to keep it in the geometry class.
After loading a geometry we can run the cleanup.